tests/virtualization/universal/download_image.pm

```diff
@@ -0,0 +1,36 @@
+#Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Download disk image
+# Maintainer: Pavel Dostal <pdostal@suse.cz>
+
+use base "consoletest";
+use virt_autotest::common;
+use virt_autotest::utils;
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+
+    ensure_default_net_is_active();
+
+    assert_script_run "if [ ! -d \"/mnt/pandora\" ]; then mkdir /mnt/pandora; fi";
+    assert_script_run "mount pandora.suse.cz:/var/lib/libvirt/images /mnt/pandora";
```

Comment:
What do you think about moving `pandora.suse.cz:/var/lib/libvirt/images` to a SETTING instead of having hard-coded configuration in here?

I suggest something like `VIRT_IMAGE_PATH` and replace `/mnt/pandora` with `/mnt/virt_images`:

    assert_script_run "if [ ! -d \"/mnt/virt_images\" ]; then mkdir /mnt/virt_images; fi";
    assert_script_run "mount " . get_var('VIRT_IMAGE_PATH') . " /mnt/virt_images";